src/content/architecture/service-discovery/_index.en.md

```diff
@@ -4,38 +4,43 @@ date: 2020-03-02T21:25:35+01:00
+The Lighthouse Agent runs in every cluster and accesses the Kubernetes API server running in
+the broker cluster to exchange service metadata information with other clusters. Local service
+information is exported to the broker and service information from other clusters is imported.
 
-#### WorkFlow
+#### Workflow
 The workflow is as follows:
 
 - Lighthouse agent connects to the broker's K8s API server.
--  For every service in the local cluster, it creates a corresponding MultiClusterService resource and exports it to the broker to be consumed by other clusters.
-- For every MultiClusterService resource in the broker exported from another cluster, it creates a copy of it in the local cluster.
+-  For every service in the local cluster, it creates a corresponding MultiClusterService resource
+and exports it to the broker to be consumed by other clusters.
+- For every MultiClusterService resource in the broker exported from another cluster, 
+it creates a copy of it in the local cluster.
 
 ![Lighthouse Agent WorkFlow](/images/lighthouse/controllerWorkFlow.png)
 <!-- Image Source: /images/lighthouse/source/controllerWorkFlow.vsdx  -->
 
-### Lighthouse Plugin
-Lighthouse plugin can be installed as an external plugin for CoreDNS, and will work along with the default Kubernetes plugin. It uses the MultiClusterService resources that are distributed by the controller for DNS resolution. The below diagram indicates a high-level architecture.
+### Lighthouse DNS Server
+The Lighthouse DNS server runs as an external DNS server which owns the domain supercluster.local.
+KubeDNS is configured to forward any request sent to supercluster.local to the Lighthouse DNS server,
+which uses the MultiClusterService resources that are distributed by the controller for DNS resolution.
 
-![Lighthouse Plugin Architecture](/images/lighthouse/lighthousePluginArchitecture.png)
-
-#### WorkFlow
+#### Workflow
 The workflow is as follows.
 
-- A pod tries to resolve a Service name.
-- The Kubernetes plugin in CoreDNS will first try to resolve the request. If no records are present the request will be sent to Lighthouse plugin.
-- The Lighthouse plugin will use its MultiClusterService cache to try to resolve the request.
-- If a record exists it will be returned, otherwise the plugin will pass the request to the next plugin registered in CoreDNS.
+- A Pod tries to resolve a Service name using the domain name supercluster.local 
+- KubeDNS forwards the request to the Lighthouse DNS server.
+- The Lighthouse DNS server will use its MultiClusterService cache to try to resolve the request.
+- If a record exists it will be returned, else an NXDomain error will be returned.
 
 ![Lighthouse CoreDNS WorkFlow](/images/lighthouse/coreDNSWorkFlow.png)
```

Comment:
Woops we keep referencing coreDNSWorkFlow.png here but the image has been deleted.

Reply:
ohh, sorry not deleted, just reduced in size as you said on slack